src/backend/protocols/go.imap/cmd/imapworker/cli_cmds/start.go

```diff
@@ -131,3 +153,12 @@ func readConfig(config *CmdConfig) error {
 	config.LDAConfig.AppVersion = __version__
 	return nil
 }
+
+func randomIdentifier() string {
+	var buf [4]byte
+	_, err := io.ReadFull(rand.Reader, buf[:])
+	if err != nil {
+		panic(err)
```

Comment:
panic is may too violent, no ?

Reply:
panic removed, a default string is returned instead (commit #f4081d58)